release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2514,6 +2514,37 @@ $ oc annotate -n <NAMESPACE> route/<ROUTE-NAME> "haproxy.router.openshift.io/bal
 
 * If the images for {op-system} and the Machine Config Operator (MCO) do not change during an upgrade from an {product-title} 4.8.z release to a later 4.8.z release, the upgrade is marked as complete before the control plane nodes have finished upgrading. Following this, the upgrade might fail if you perform operations on your cluster before the upgrade is actually complete. As a workaround, verify that updates are completed on the control plane nodes before performing additional operations on the cluster. You can use the `oc get mcp/master` command to review the status of the MCO-managed nodes that are available on your cluster for each pool. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2025396[*BZ#2025396*])
 
+* After upgrading from a 4.7 {product-title} cluster to 4.8, the routing path from pod internal network to external network through secondary network interface controllers (NICs) on {product-title} nodes is disabled by default. This is due to the shared gateway being the default gateway mode in Open Virtual Network (OVN) design starting from 4.8. If that routing path is required, as a workaround before or after the upgrade, create a `gateway-mode-config` config map in the `openshift-network-operator` namespace to force OVN Gateway mode to local.
```

Comment:
```suggestion
* After upgrading from a 4.7 {product-title} cluster to 4.8, the routing path from pod internal network to external network through secondary network interface controllers (NICs) on {product-title} nodes is disabled by default. This is due to the default gateway mode in Open Virtual Network (OVN) design starting from 4.8. If that routing path is required, as a workaround before or after the upgrade, create a `gateway-mode-config` config map in the `openshift-network-operator` namespace to force OVN Gateway mode to local.
```

Reply:
I was trying to reword for clarity. Not sure if it helps. I think another way to do reword this would be: This is due to the default gateway mode being (set to?) shared in Open Virtual Network (OVN) design from 4.8.